packages/core/src/toast/Toast.css

```diff
@@ -28,12 +28,9 @@
 }
 
 /* Styles applied to icon */
-.saltToast-icon.saltIcon {
+.saltToast-iconContainer > .saltIcon {
   margin: calc(var(--salt-spacing-75) + 2px) 0 var(--salt-spacing-75) 0;
-}
-
-.salt-density-high.saltToast-icon.saltIcon {
-  margin: calc(var(--salt-spacing-75) + 1px) 0 var(--salt-spacing-75) 0;
+  color: var(--toast-borderColor);
```

Comment:
Just to note that we're applying border color to foreground of icon here. May need to revisit again in the future.

Reply:
can change to  
```
  --toast-statusColor: var(--salt-container-primary-borderColor);
  --toast-borderColor: var(--toast-statusColor);
```
and use ` --toast-statusColor` for the icon, thoughts?